### PR TITLE
Issue #42 detection logic changed

### PR DESCRIPTION
**Pull request by p-a-c-a**
Detection logic for available Vermittlungscodes changed to solve issue #42. Message added in booking.ts that automatic Vermittlungscode booking is not supported.

### CODE REVIEW

**Comment by patrick246** [comment]
From a quick glance, it seems our fixes are compatible :)

File: src/appointments.ts [hunk 1]
```diff
@@ -148,20 +148,30 @@ export async function proceedWithoutACode(page: Page) {
   }
   debug("Waiting 20s for appointments alert to show");
   await page.waitForTimeout(20000);
-
-  const appointmentWarning = await page.$("div.alert.alert-danger");
+  
+  // detect available codes by checking if the "Schnellprüfung durchführen" Button exists
+  var schnellpruefungOK = false;
```
Comment (marcoklein) [inline src/appointments.ts:153]: Please use typescript `let`.

File: .gitignore [hunk 2]
```diff
@@ -3,4 +3,5 @@ build
 .env
 *.log
 config.json
-dist
\ No newline at end of file
+dist
+config - StandardTest.json
```
Comment (marcoklein) [inline .gitignore:7]: Please remove custom ignore.

**Review by marcoklein — COMMENTED** [review]
Thanks for the PR - would like to merge that as soon as possible - could you look into my comments please?

**Pull request closed without merging**